includes/class-webmention-receiver.php

```diff
@@ -96,6 +133,15 @@ public static function register_meta() {
 			'show_in_rest' => true,
 		);
 		register_meta( 'comment', 'webmention_response_code', $args );
+
+		// Purpose of this is to store a vouch URL
```

Comment:
Can you move this to the vouch class, otherwise it will load, if vouch is enabled or not...

Reply:
I'd like to store the vouch URL whether or not vouch is enabled or not, in the event it becomes stable in future.